modules/ossm-threescale-istio-adapter-troubleshooting.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v1x/threescale_adapter/threescale-adapter.adoc
+// * service_mesh/v2x/threescale_adapter/threescale-adapter.adoc
+
+[id="ossm-threescale-istio-adapter-troubleshooting_{context}"]
+= 3scale Istio adapter troubleshooting
+
+You might want to check whether the 3scale Istio adapter is working as expected. If this is not the case, you can troubleshoot from the list of scenarios.
+
+Perform the following steps to check that the adapter is working correctly:
+
+. Ensure the _3scale-adapter_ pod is running in the control plane namespace, that is `istio-system`.
```

Comment:
Remember that `istio-system` is just a default choice for this namespace, so the user might have chosen a different name or even have multiple versions of Service Mesh installed in different namespaces.